core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -641,6 +643,24 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     }
   }
 
+  def sendChangeNotification(workspaceName: WorkspaceName): Future[PerRequestMessage] = {
+
+    def getEmails = {
+      dataSource.inTransaction{ dataAccess =>
+        withWorkspaceContext(workspaceName, dataAccess) {workspaceContext =>
+          dataAccess.workspaceQuery.listEmailsAndAccessLevel(workspaceContext)
+        }
+      }
+    }
+
+    for {
+      emails <- getEmails
+    } yield {
+      val notificationMessages = emails.map(_._1).map(email => Notifications.WorkspaceChangedNotification(email, workspaceName))
```

Comment:
as you are probably finding, email is not what you want. You want the subject id for each user. WorkspaceChangedNotification needs to have a recipientUserId member and that should be the subject id